docs/sidebars.js

```diff
@@ -44,14 +44,19 @@ const sidebars = {
       label: "SQL Reference",
       link: { type: "doc", id: "sql/intro" },
       items: [
-        "sql/types",
-        "sql/boolean",
-        "sql/comparisons",
-        "sql/integer",
-        "sql/float",
-        "sql/decimal",
-        "sql/fp",
-        "sql/string",
+          "sql/grammar",
```

Comment:
was this formatted using `yarn format`?

Reply:
no.
who remembers what tools one needs to use?
it should be part of the commit process if it's necessary

Reply:
It's documented in the readme but I'll add it to CI